src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceAfterCheck.java

```diff
@@ -170,8 +170,7 @@ public void setAllowLineBreaks(boolean allowLineBreaks) {
     public void visitToken(DetailAST ast) {
         final DetailAST whitespaceFollowedAst = getWhitespaceFollowedNode(ast);
 
-        if (whitespaceFollowedAst.getNextSibling() == null
-                || whitespaceFollowedAst.getNextSibling().getType() != TokenTypes.ANNOTATIONS) {
+        if (!ignoreWhitespaceAfter(whitespaceFollowedAst)) {
```

Comment:
Can we make this condition a positive (do away with `!`)?

Reply:
for sure, I will push a change